packages/terra-button/src/Button.jsx

```diff
@@ -101,6 +101,16 @@ const propTypes = {
    * Sets the button variant. One of `neutral`,  `emphasis`, `ghost`, `de-emphasis`, `action` or `utility`.
    */
   variant: PropTypes.oneOf([ButtonVariants.NEUTRAL, ButtonVariants.EMPHASIS, ButtonVariants.GHOST, ButtonVariants['DE-EMPHASIS'], ButtonVariants.ACTION, ButtonVariants.UTILITY]),
+  /**
+   * @private
+   * Whether or not the button can be selected (toggleable button).
+   */
+  isSelectable: PropTypes.bool,
```

Comment:
can we add a console.warn when this prop s used ? something like 'this prop will be deprecated in future and do not recommend it for use'.

Reply:
Added in https://github.com/cerner/terra-core/pull/3940/commits/3ea7af9fff4dee8f74d7bd55475473c52bfd2265

Reply:
Removed in https://github.com/cerner/terra-core/pull/3940/commits/44abd39cb3a184cee8fb42c411ed0b9ee6bf494d as prop was changed to custom.